examples/benchmarks/bubblebench/simple-tree/src/appState.ts

```diff
@@ -0,0 +1,70 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import {
+	type IAppState,
+	type IBubble,
+	type IClient,
+	makeBubble,
+	randomColor,
+} from "@fluid-example/bubblebench-common";
+import type { TreeView } from "@fluidframework/tree/internal";
```

Comment:
FYI, please use the public (not /internal) path for public types.
There is no enforcement yet, but so just keep an eye out.
cc @taylorsw04